eldritch/values.py

```diff
@@ -1,12 +1,17 @@
 import abc
 import collections
+import math
 import operator
 
 from eldritch import places
 
 OPER_MAP = {"at least": operator.ge, "at most": operator.le, "exactly": operator.eq}
 
 
+def ceildiv(lhs, rhs):
```

Comment:
Seems simper to have a single operator rather than a chain of Calculations for rounding up.